### ISSUE TITLE
Error during training

### ISSUE THREAD

**Issue by dergo853** [lightvector/KataGo#652]
When I trained KataGo, it made a problem:
Shuffled data path does not exist, there seems to be no shuffled data yet, waiting and trying again later: /content/KataGo/python/selfplay/shuffleddata/current
How can I solve it?

**Comment by lightvector** [comment]
Presumably you are running it with command line arguments such that it expects the shuffled data to be at /content/KataGo/python/selfplay/shuffleddata/current

Is that the path you intended? If not, you need specify the correct path, presumably the path that you are actually outputting the shuffled data to via shuffle.py or shuffle.sh.

If this is the very start of training from scratch, there might not be enough data yet, which is why it says it is waiting and trying again later. Check all the logs and paths for your selfplay processes and your shuffle processes.

**Comment by dergo853** [comment]
The parameters I set are 
./selfplay/train.sh selfplay b6c96 b6c96 256 main -lr-scale 1.0 >> log.txt 2>&1 & disown

**Comment by lightvector** [comment]
Okay, but where is the data that you are trying to train on?

**Comment by dergo853** [comment]
They are npz files, right?

**Comment by lightvector** [comment]
Great, yes, npz files are the raw data, and assuming you're using the current training code that uses Tensorflow, the shuffled data should be tfrecord files. So where is your shuffle script outputting the shuffled data? Check the logs for your shuffle script.

**Comment by dergo853** [comment]
Logs are in log.txt?

**Comment by lightvector** [comment]
I have no idea, it depends on what file paths you're using. Are you shuffling the data at all?
https://github.com/lightvector/KataGo/blob/master/python/selfplay/shuffle.sh

**Comment by lightvector** [comment]
And:
https://github.com/lightvector/KataGo/blob/master/python/selfplay/shuffle_loop.sh

**Comment by dergo853** [comment]
My npz path is correct.  I am a newbie, ha.

**Comment by lightvector** [comment]
If you have the raw data already, then you need to run a shuffle script that will take the raw data and shuffle it and convert it to tfrecord files and output it into shuffleddata/current, *then* the training script will be able to find it the data it needs in shuffleddata/current.

I'm going afk shortly, so I can't help you any further, but if you still need help, please go to https://discord.gg/3jfxmrSqgC and ask there.
